chingu-voyage5/Bears-Team-3
# Connect Front-end, Back-end and Database

Closes #4 

# Changes
* Combine `.gitignore` files
* Remove unused `package.json` files
* Add proxy
* Prove proxy works in Vue by writing button that calls Ruby back-end

# To Test
1. Stand-up Rails server locally
2. Stand-up Vue dev server locally
3. Go to `http://localhost:8080`
4. Confirm that clicking the button increases counter
5. Confirm that the counter persists when refreshing browser
